core/server/data/importer/index.js

```diff
@@ -102,22 +102,18 @@ _.extend(ImportManager.prototype, {
      * @returns {Function}
      */
     cleanUp: function () {
-        var filesToDelete = this.filesToDelete;
-        return function (result) {
```

Comment:
Looks like `cleanUp` wasn't functioning properly because it was returning a callable function for some reason... which would have to be used as `cleanUp()(result)`.  Your change seem to make sense :+1: